### ISSUE TITLE
Customize the placeholder message when composing the message

### ISSUE THREAD

**Issue by mfrohman123** [wiredashio/wiredash-sdk#359]
**Is your feature request related to a problem? Please describe.**
It would be nice to be able to change the placeholder message of the text entry on the first feedback screen. The current message may not be relevant to application context, i.e. it may only accept feature requests/not have avatars.

**Describe the solution you'd like**
A way to customize the placeholder message, such as an option in WiredashFeedbackOptions or an option in Wiredash.show.

<img width="989" alt="image" src="https://github.com/user-attachments/assets/bd62bf70-7484-469b-ad5e-1006d313b350">

The application this is used for in the image above does not have avatars and supports feedback for feature requests in addition to bugs.

**Comment by tmaxxdd** [comment]
Yeah especially in Polish this is misleading user as he/she sees "Opisz krótko problem, który widzisz", what means "Describe the problem you see". 

This is wrong placeholder for FEEDBACK type of user's response
